core/invoker/src/main/scala/whisk/core/containerpool/ContainerPool.scala

```diff
@@ -178,6 +178,9 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
     case RescheduleJob =>
       freePool = freePool - sender()
       busyPool = busyPool - sender()
+
+    case Busy =>
+      sender ! (busyPool.size != 0)
```

Comment:
Should be `sender ! busyPool.nonEmpty`

It's valuable to make it a habit to replace all `size > 0` and `size == 0` with the respective `nonEmpty` or `isEmpty` equivalents. It can be very costly (require a whole iteration and thus be `O(n)`) to calculate the size of a datastructure while `isEmpty`/`nonEmpty` can always be calculated in `O(1)`

Doesn't matter here, but a good habit to foster.